src/averageAges.js

```diff
@@ -15,6 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  let man = [];
+
+  man = people.filter((person) => person.sex === 'm');
+
+  if (century) {
+    man = man.filter((person) => Math.ceil(person.died / 100) === century);
+  }
```

Comment:
there is no sense to use filter twice

your algorithm will work faster if you iterate just once: 
  imagine you have array with n elements, when you iterate throw it you do n operations, so when you iterate throw it twice you will have 2n operations - > less operations === faster code -> n will be faster then 2n 
  
you can add our condition to filter function: 
```suggestion
    man = people.filter((person) => century 
      ?  Math.ceil(person.died / 100) === century) && person.sex === 'm' 
      :  person.sex === 'm' 
    );
```